src/components/dashboard/Usage.vue

```diff
@@ -51,70 +51,89 @@ const allLimits = computed(() => {
 })
 
 async function getAppStats() {
-  const date_id = new Date().toISOString().slice(0, 7)
   if (!main.user)
     return { data: [], error: 'missing user' }
 
   if (props.appId) {
-    // console.log('appID', props.appId)
     return supabase
-      .from('app_stats')
+      .from('app_usage')
```

Comment:
get the app_usage with mode = day, and created_at between the cycle limit.

Reply:
I've added the mode=day filter, but can't get between the cycle limit as right now we need to increment whatever's there in created_at to see some data coming in.